cmd/minikube/cmd/start.go

```diff
@@ -949,3 +954,26 @@ func extractVMDriverVersion(s string) string {
 	v := strings.TrimSpace(matches[1])
 	return strings.TrimPrefix(v, version.VersionPrefix)
 }
+
+func validateOSSupportVMDriver(os, vmDriver string) bool {
+	switch vmDriver {
+	case constants.DriverNone:
```

Comment:
Rather than duplicating the platform-to-driver map, could this be adjusted to use the new build-tag specific list?